documentation/staging/content/userguide/managing-operators/conversion-webhook.md

```diff
@@ -0,0 +1,112 @@
+
+{{% notice note %}}
+The cluster-level permissions for listing and reading the Namespaces and Deployments are needed to search all the namespaces for existing conversion webhook deployment. In addition, permissions to create and read the events in the conversion webhook namespace are needed for the conversion webhook service account.
+{{% /notice %}}
+
+
+If you want to **only** install the conversion webhook (and not the Operator) in the given namespace, set the Helm configuration value `webhookOnly` to `true` in the `helm install` command. After meeting the [prerequisite requirements]({{<relref "/userguide/managing-operators/preparation.md">}}), call:
+```
+$ helm install sample-weblogic-conversion-webhook \
+  weblogic-operator/weblogic-operator \
+  --namespace sample-weblogic-conversion-webhook-ns \
+  --set "webhookOnly=true" \
+  --wait
+```
+The above command creates a Helm release named `sample-weblogic-conversion-webhook` in the `sample-weblogic-conversion-webhook-ns` namespace, configuring a deployment and supporting resources for the conversion webhook and deploying the conversion webhook.
+
+To check if the conversion webhook is deployed and running,
+see [Troubleshooting]({{<relref "/userguide/managing-operators/troubleshooting#check-the-conversion-webhook-deployment">}}).
+
+To prevent the removal of conversion webhook during the `helm uninstall` command, set the Helm configuration value `preserveWebhook` to `true` during installation using the `helm install` command. 
+
+The following table describes the behavior of different `Helm` commands (with the Operator Helm chart) and the Helm configuration values:
+
+| Helm command with the operator Helm chart | Helm configuration values | Behavior |
+| --- | --- | --- |
+| Helm install | Default | Operator and conversion webhook installed. |
+| Helm install | Default and existing `weblogic-operator-webhook` deployment | Operator installed. |
+| Helm install | `webhookOnly=true` | Conversion webhook installed. |
+| Helm install | `preserveWebhook=true` | Operator and webhook installed. |
+| Helm uninstall | | Operator and webhook deployment uninstalled. |
+| Helm uninstall with `preserveWebhook=true` set during `helm install` | | Operator deployment uninstalled (and webhook deployment preserved). |
+
+#### Uninstall the conversion webhook
+When you [uninstall the operator]({{<relref "/userguide/managing-operators/installation#uninstall-the-operator">}}) using the `helm uninstall` command, it removes the conversion webhook 
+associated with the release and its resources from the Kubernetes cluster. However, if you **only** installed the conversion webhook using `webhookOnly=true` Helm configuration value, then run the `helm uninstall` 
+command separately to remove the conversion webhook associated with the release and its resources.
```

Comment:
IMO this section could use a "Note", or at least an explanation, about why you may not want to uninstall the webhook  when uninstalling aan operator (if you have other operators, or want to continue being able to deploy domains even when an operator isn't installed).